.coveragerc

```diff
@@ -1,7 +1,23 @@
 [run]
+source_pkgs =
+	setuptools
+	pkg_resources
 omit =
 	# leading `*/` for pytest-dev/pytest-cov#456
-	*/.tox/*
+	*/_vendor/*
+	*/_distutils/*
```

Comment:
Currently most of `_distutils` shows up as 0 hit. I wonder if I should not remove it...

Reply:
My guess is it shows up as not being hit because of the name mismatch (`setuptools._distutils` versus `distutils`). I do try to avoid these snowflake configurations, but I also recognize they may be needed.

Reply:
@abravalheri I wonder if just Codecov needs to be configured not to take it into account vs the underlying coveragepy.

Reply:
Thank you @jaraco and @webknjaz for the review.

> I wonder if just Codecov needs to be configured not to take it into account vs the underlying coveragepy.

My approach would be trying to solve things with coveragepy first, because on my personal machine I rely on the terminal report which is very useful. Any Codecov change would just be useful after a PR is made...